CHANGELOG.rst

```diff
@@ -167,3 +167,186 @@ Commits
   https://github.com/fedora-infra/fedimg/commit/f891dccc9
 - ddbb82523 Remove the spec file.  We keep it in Fedora dist-git.
   https://github.com/fedora-infra/fedimg/commit/ddbb82523
+
+
+0.6
+---
+
+General
```

Comment:
You could optionally mark these as sub-sections with something like ``^^^^`` or ``~~~~``. I think RST is pretty flexible as long as you are consistent.